Week-11/redux-exercise-2/src/redux/reduser.js

```diff
@@ -0,0 +1,16 @@
+import { actions } from "./actions";
+
```

Comment:
check for all the typos, its "reducers"

Also give proper file names.

Reply:
Solved